modules/distribution-zones/src/test/java/org/apache/ignite/internal/distributionzones/DistributionZoneManagerTest.java

```diff
@@ -92,8 +92,6 @@ public void setUp() {
 
         registry.start();
 
-        registry.initializeDefaults();
```

Comment:
Right after I said _"they are always called together"_, I see that they're not. Is this intentional, or you simply forgot to call the method?